tests/unit/amo/components/TestRatingsByStar.js

```diff
@@ -148,30 +155,59 @@ describe(__filename, () => {
     const addon = addonForGrouping(grouping);
     store.dispatch(setGroupedRatings({ addonId: addon.id, grouping }));
     const root = render({ addon });
-    const counts = root.find('.RatingsByStar-star').find(Link);
-    const bars = root.find('.RatingsByStar-barContainer').find(Link);
```

Comment:
I think the description of this test should be updated, as I think it is no longer accurate. For example, it's not asserting anything about bars.

Reply:
@bobsilverberg Should I change the description or add test for bars ?

Reply:
I my opinion because we have separate test case for bars, we can change the description of the test. So, after merging star count test description will be `renders star labels, star counts and links`.

Reply:
@bobsilverberg Thanks!